monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -33,15 +33,18 @@ sealed trait Rx[+A] { self =>
    * `for { a <- ra; b <- rb } yield (a, b)`.
    *
    * ```
-   * // r1      => 0     8                       9     ...
-   * // r2      => 1           4     5     6           ...
-   * // product => (0,1) (8,1) (8,4) (8,5) (8,6) (9,6) ...
+   * // r1  => 0     8                       9     ...
+   * // r2  => 1           4     5     6           ...
+   * // zip => (0,1) (8,1) (8,4) (8,5) (8,6) (9,6) ...
    * ```
    *
    * This method, together with `Rx.apply`, forms am `Applicative`.
    * `|@|` syntax is available via the `monadic-rx-cats` package.
    */
-  def product[B](other: Rx[B]): Rx[(A, B)] = Product[A, B](this, other)
+  def zip[B](other: Rx[B]): Rx[(A, B)] = Zip[A, B](this, other)
+
+  @deprecated("Renamed to zip", "0.3.3")
+  def product[B](other: Rx[B]): Rx[(A, B)] = zip(other)
```

Comment:
It should be easy to provide a scalafix rewrite for this migration. If you're interested